kuma/wiki/templates/wiki/document.html

```diff
@@ -73,6 +73,25 @@
   <meta property="og:description" content="{{ seo_summary }}">
   <meta name="description" content="{{ seo_summary }}">
   <meta name="twitter:description" content="{{ seo_summary }}">
+
+  <!-- Scrollback widget -->
+  <script>
+    var scrollbackAdded = false;
+  	function addScrollback(roomName) {
+      if (!roomName) return;
+      if(!scrollbackAdded) {
+    		window.scrollback = {"room": roomName,"form":"toast","minimize":false, titlebarColor: "#00539f"};
+        var el = document.createElement("script");
+        el.async = 1;
+        el.src = (location.protocol === "https:" ? "https:" : "http:") + "//scrollback.io/client.min.js";
+        document.getElementsByTagName("script")[0].parentNode.appendChild(el);
+        scrollbackAdded = true;
+      } else {
+        var iframe = document.getElementsByClassName("scrollback-stream")[0];
+        iframe.src = iframe.src.replace(/\/[^/]*\?/, '/' + encodeURIComponent(roomName) + '?');
+      }
+  	}
+  </script>
```

Comment:
@amalantony we put JS files in [`media/redesign/js`](https://github.com/mozilla/kuma/tree/master/media/redesign/js)

Then, we add a reference to the file under the appropriate section of [the `MINIFY_BUNDLES['js']` element in `settings.py`](https://github.com/mozilla/kuma/blob/master/settings.py#L710-L784).

This template extends the `wiki/base.html` template, [which includes the '`wiki`' script bundle](https://github.com/mozilla/kuma/blob/master/kuma/wiki/templates/wiki/base.html#L13-L15), so ...

* [ ] Add a `scrollback.js` file to `media/redesign/js`
* [ ] Add `redesign/js/scrollback.js` to the `wiki` section of `MINIFY_BUNDLES['js']`

Does that make sense?